src/test/crimson/perf_async_msgr.cc

```diff
@@ -124,5 +124,7 @@ int main(int argc, char** argv)
                          CODE_ENVIRONMENT_UTILITY,
                          CINIT_FLAG_NO_MON_CONFIG);
   common_init_finish(cct.get());
+  cct->_conf.set_val("ms_crc_header", "false", nullptr);
+  cct->_conf.set_val("ms_crc_data", "false", nullptr);
```

Comment:
but in real world, these options are enabled. if we want to understand how crc digest hurts the performance of messenger, probably we can just an an option, instead of disabling them.

again, please elaborate the rationale of this change in the commit message.

Reply:
added options to both `perf-crimson-msgr` and `perf-async-msgr`, and also added commit message.